database/frontend/design-inspirations.json

```diff
@@ -130,5 +130,20 @@
         "url": "https://screenlane.com/",
         "category": "frontend",
         "subcategory": "design-inspiration" 
-    }
-]
\ No newline at end of file
+    },
+     {
+        "name": "Screenlane",
+        "description": "You can find app screens, landing pages, user flow inspirations, and keep up with the latest mobile UI design trends.",
+        "url": "https://screenlane.com/",
+        "category": "frontend",
+        "subcategory": "design-inspiration" 
+
+    },
+    {
+        "name": "Design Vault",
+        "description": "Design Vault is a website that showcases designs from real products, giving you access to the latest and greatest in UI patterns and trends.",
+        "url": "https://designvault.io/",
+        "category": "frontend",
+        "subcategory": "design-inspiration"
+     }
+]
```

Comment:
```suggestion
}
```

Reply:
> 

ok I have put curly bracket on line 149
now you can review it